docs/functions/add.md

```diff
@@ -0,0 +1,17 @@
+## add
```

Comment:
If we moved the proper Haskell notation from Table of Contents to here, I think it would look better.